views/forms/UserProfileForm/UserProfileFormFields.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import TextInputTopLabel from '../TextInputTopLabel'
+
+import { borderRadius } from '../../../constants'
+
+import './UserProfileFormFields.css'
+
+const UserProfileFormFields = ({ control }) => {
+  return (
+    <div className="UserProfileFormInputs_box">
+      <TextInputTopLabel
+        control={control}
+        sx={{
+          mb: '20px',
+        }}
+        label="Full name"
+        name="display_name"
+        inputProps={{ sx: { borderRadius: borderRadius[8] } }}
+        size="small"
+        required
+        fullWidth
+      />
+      <TextInputTopLabel
+        control={control}
+        sx={{
+          mb: '20px',
+        }}
+        label="Title and institution (optional)"
+        name="title"
+        inputProps={{ sx: { borderRadius: borderRadius[8] } }}
+        size="small"
+        fullWidth
+      />
+      <TextInputTopLabel
+        control={control}
+        inputProps={{
+          sx: {
+            borderRadius: borderRadius[8],
+          },
+        }}
```

Comment:
should these styles be applied in the `TextInputTopLabel` directly so we don't need to pass them every time?

Reply:
[2b10634](https://github.com/AMP-SCZ/dpdash/pull/478/commits/2b10634c506de607903b3c18f8cbfd56e8904186)